Lesson5/Task1/Rectangle.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Lesson5
+{
+    class Rectangle
+    {
+        private int side1, side2;
```

Comment:
@okuzmina It's extra vars, your public properties Side1 and Side2 that you are using will create "hided" private fields, and you are not using your private side1 and side2 anywhere, could be removed